mgmt/rest/v2/mock/mock_task_manager.go

```diff
@@ -179,7 +179,7 @@ const (
       "deadline": "4ns",
       "creation_timestamp": -62135596800,
       "last_run_timestamp": -1,
-      "task_state": "Running",
+      "state": "Running",
```

Comment:
changed `task_state` to `state` so that the JSON representation(currently it's `task_state`) is same as the variable name (`state`). The reason is:
* go-swagger prefers the variable name same as its JSON representation.
* `task_state` only appears in the Snap response but not any input. It's not a breaking change.
* `state` is inside `task` struct. The name `task_state` is redundant.